src/components/HomeView.jsx

```diff
@@ -21,15 +21,17 @@ const HomeView = () => {
       userData.email,
       userData.password
     );
-    if(response.status === 'success'){
-      dispatch({ type: 'SET_WELCOME_MESSAGE', payload: `Welcome ${response.data.name}!` });
-      dispatch({ type: 'SET_CURRENT_USER', payload: response.data });
-      navigate('dashboard', {
-        state: { flash: `Welcome ${response.data.name}!`, currentUser: response.data },
+    if (response.status === "success") {
+      dispatch({ type: "SET_CURRENT_USER", payload: response.data });
+      navigate("dashboard", {
+        state: {
+          flash: `Welcome ${response.data.name}!`,
+          currentUser: response.data
```

Comment:
Now that you have redux, I believe relying on this `react-router` hook is duplicating passing the state around, especially since it looks like you are now using the `useSelector` imported from redux in `Dashboard.jsx`. Try to see if you can remove this.

Reply:
Ops, that's old code, I forgot about it